app/controllers/search_controller.rb

```diff
@@ -12,6 +14,18 @@ def suggest
       user_results = user_results.decorate
     end
 
-    render partial: 'suggest', locals: { users: user_results || [], groups: group_results || [] }
+    render partial: 'suggest', locals: { users: user_results || [], groups: group_results || [], attributes: :room }
+  end
+
+  def suggest_leader
+    if params.key?(:query)
+      count = Rails.configuration.x.results_per_page
+      offset = count * params[:user_count].to_i
+      user_results = SearchQuery.new.user_search(params[:query], offset, count)
+      user_results = user_results.decorate
```

Comment:
Can this be written in one line?

Reply:
Fixed it. I also tried to improve the naming a little.